library/src/containers/Schemas/SchemaProperties.tsx

```diff
@@ -45,12 +46,32 @@ const schemaPropertiesAccessors: Array<TableAccessor<SchemaElement>> = [
   ),
   el => <span>{el.schema.content.format}</span>,
   el => <span>{el.schema.content.default}</span>,
-  el =>
-    el.schema.content.description && (
-      <Markdown>{el.schema.content.description}</Markdown>
-    ),
+  el => {
+    const enumElements = getEnumHTMLElements(el.schema);
+    return (
+      el.schema.content.description && (
+        <div>
+          <Markdown>{el.schema.content.description}</Markdown>
+          {enumElements.length && <div>Enum: {enumElements}</div>}
```

Comment:
I found a problem with conditional rendering in this place.

For example. If we have schema like that (this is entry of schemas field):

```yaml
    turnOnOffPayload:
      type: object
      properties:
        command:
          type: string
          enum:
            - on
            - off
          description: Whether to turn on or off the light.
        sentAt:
          $ref: "#/components/schemas/sentAt"
```

Then by your code, rendered enums look like:
![image](https://user-images.githubusercontent.com/20404945/80383767-55940700-88a4-11ea-94c5-a857a3ea4fc0.png)

According to React docs https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator conditional rendering in React only avoid false, null and undefined values. The other values (also 0 - second row) are render - if condition return false. So your code for return enums should looks like:

```js
{enumElements.length > 0 && <div>Enum: {enumElements}</div>}
```

or

```js
{enumElements.length ? <div>Enum: {enumElements}</div> : null}
```

I also wonder if the description place is good for rendering enums. Better would be default, but since you had to wait a long time for review, we can leave it.

Reply:
Hej @magicmatatjahu no worries. Thanks a lot. I changed the code like you mentioned above. Thanks for the input ;) I hope I didn't make any pressure :D